Src/Test.UnitTests.Sarif.PatternMatcher/SpamTestRule.cs

```diff
@@ -29,7 +29,9 @@ internal class SpamTestRule : Skimmer<AnalyzeContext>, IOptionsProvider
         internal static Random s_random = new Random(s_seed);
 
         [ThreadStatic]
+#pragma warning disable CS0649 // Field 'SpamTestRule.s_testRuleBehaviors' is never assigned to, and will always have its default value
         internal static TestRuleBehaviors s_testRuleBehaviors;
+#pragma warning restore CS0649 // Field 'SpamTestRule.s_testRuleBehaviors' is never assigned to, and will always have its default value
```

Comment:
>s_testRuleBehaviors [](http://example.com/codeflow?start=54&length=19)

Just an opinion, you don't need to repeat this text since I can see it a mere two lines away. :)

Reply:
I actually wanted to see what made sense to do with this field. It doesn't seem to be used anywhere.
And the #pragma text is automatic. Not sure what you mean about repeating this text...